massa-execution-worker/src/execution.rs

```diff
@@ -1294,6 +1311,19 @@ impl ExecutionState {
 
         // Broadcast a final slot execution output to active channel subscribers.
         if self.config.broadcast_enabled {
+            let slot_exec_out = SlotExecutionOutput::ExecutedSlot(exec_out.clone());
```

Comment:
After further looking at the code base this one is a duplicate because it will already be sent by the call L.1301 to `execute_slot`

Reply:
This is in case we keep the actual behaviour. I'll correct